azkaban-exec-server/src/main/java/azkaban/execapp/AzkabanExecutorServer.java

```diff
@@ -94,6 +94,19 @@
   private final ArrayList<ObjectName> registeredMBeans = new ArrayList<ObjectName>();
   private MBeanServer mbeanServer;
 
+  public static boolean testing = false;
+
+  public AzkabanExecutorServer(ExecutorLoader executionLoader, FlowRunnerManager runnerManager, Props props,
```

Comment:
I wonder if with the recent introduction of Guice. There is a better way to construct different dependencies in tests so that we can avoid introducing this new test only constructor.

@suvodeep-pyne